pkg/cli/prompt/text/text_test.go

```diff
@@ -78,7 +78,7 @@ func Test_E2E(t *testing.T) {
 			Placeholder: "test placeholder",
 		}
 		model := NewTextModel("test prompt", options)
-		return teatest.NewTestModel(t, model, teatest.WithInitialTermSize(80, 50))
+		return teatest.NewTestModel(t, model, teatest.WithInitialTermSize(18, 50))
```

Comment:
The problem is happening here. Because the width is set to 80, we see the extra spaces in the actual output.

```
=== RUN   Test_E2E/cancel
    /Users/yetkintimocin/dev/msft/radius-project/radius/pkg/cli/prompt/text/text_test.go:92: Testing output:

    /Users/yetkintimocin/dev/msft/radius-project/radius/pkg/cli/prompt/text/text_test.go:92: Testing output:

        test prompt

        > test placeholder

        (ctrl+c to quit)
    /Users/yetkintimocin/dev/msft/radius-project/radius/pkg/cli/prompt/text/text_test.go:159:
        	Error Trace:	/Users/yetkintimocin/dev/msft/radius-project/radius/pkg/cli/prompt/text/text_test.go:159
        	Error:      	Not equal:
        	            	expected: "test prompt\n\n> test placeholder\n\n(ctrl+c to quit)"
        	            	actual  : "test prompt\n\n> test placeholder                         \n\n(ctrl+c to quit)"

        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -2,3 +2,3 @@

        	            	-> test placeholder
        	            	+> test placeholder
```

@rynowak

Reply:
Is it adding extra spaces to meet the expected width?

Can you figure out if this is intended behavior from bubbletea or something we're doing wrong?

Reply:
@ytimocin did you get a chance to confirm this?

Reply:
I will test this today @rynowak @kachawla

Reply:
I think this is the change that broke our unit tests: https://github.com/charmbracelet/bubbles/pull/451.

Reply:
Looks like a bugfix upstream. I think updating our test is the right thing to do then 👍